pmultiqc/modules/core/core.py

```
@@ -78,12 +78,22 @@ def __init__(self):
 
         # quantms, DIA-NN, and mzid results
-        QuantMSModule = get_module("quantms", "QuantMSModule")
-        QuantMSModule(
-            self.find_log_files,
-            self.sub_sections,
-            heatmap_color_list
-        )
+        if config.kwargs.get("mzid_plugin", False):
+            # Use MzIdentMLModule for mzid plugin
+            MzIdentMLModule = get_module("mzidentml", "MzIdentMLModule")
+            mzid_module = MzIdentMLModule(
```

Ruff (0.13.1)

Local variable `mzid_module` is assigned to but never used

Remove assignment to unused variable `mzid_module`

(F841)